lldb/source/Interpreter/OptionValueFileSpecList.cpp

```diff
@@ -41,6 +41,16 @@ void OptionValueFileSpecList::DumpValue(const ExecutionContext *exe_ctx,
   }
 }
 
+llvm::json::Value
+OptionValueFileSpecList::ToJSON(const ExecutionContext *exe_ctx) {
+  std::lock_guard<std::recursive_mutex> lock(m_mutex);
+  llvm::json::Array spec_list;
```

Comment:
How about `array`, that seems more appropriate? 
```suggestion
  llvm::json::Array array;
```